### PR TITLE
Updated message to GCToolKit is a set of libraries for analyzing Hotspot

### PR DESCRIPTION
**Pull request by anantk24**
added hotspot keyword bold and italic way.
** - added for bold
one more * - added for italic
#21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Microsoft GCToolKit
 
-GCToolKit is a set of libraries for analyzing Java garbage collection (GC) log files. The toolkit parses GC log files into discrete events and provides an API for aggregating data from those events. This allows the user to create arbitrary and complex analyses of the state of managed memory in the Java Virtual Machine (JVM) represented by the garbage collection log.
+GCToolKit is a set of libraries for analyzing ***Hotspot*** Java garbage collection (GC) log files. The toolkit parses GC log files into discrete events and provides an API for aggregating data from those events. This allows the user to create arbitrary and complex analyses of the state of managed memory in the Java Virtual Machine (JVM) represented by the garbage collection log.
```
Comment (bmarwell) [inline README.md:3]: Personally, I don't think that ***this much*** emphasis is needed. ;-)

Comment (karianna) [reply README.md:3]: @anantk24 - I think just adding the word Hotspot is enough here thanks!

Comment (dsgrieve) [reply README.md:3]: @anantk24 Great add! Just to be picky, though, Hotspot should be HotSpot (with a capital S).

Comment (anantk24) [reply README.md:3]: Got it. I made those changes, because that was mentioned issue . 
I will correct and update it. Thanks for comments! :)

**Comment by anantk24** [comment]
Updated changes according to comments.

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**